src/contexts/federated.ts

```diff
@@ -1,7 +1,14 @@
 import React, { useContext } from 'react';
 import { DeleteInstanceProps } from '@rhoas/app-services-ui-shared';
 
-export type FederatedProps = Pick<DeleteInstanceProps, 'kafka'> & {
+export type KafkaRequest = Pick<DeleteInstanceProps, 'kafka'> & {
```

Comment:
While this works and this seem to be time sensitive fix. It will be good to avoid using Pick and create dedicated objects.